### PR TITLE
FLUID-5974: Updating to the forked version of jquery.selectbox

### PR DESCRIPTION
**Pull request by jobara**
The forked version has been updated to work with jQuery v3.x

https://issues.fluidproject.org/browse/FLUID-5974

### CODE REVIEW

File: tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The version of the jquery.selectbox plugin included in this folder was created from the master branch of the fluid-project fork:
+
+https://github.com/fluid-project/jquery.selectbox
```
Comment (cindyli) [inline tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md]: Can you add the link at the README of this repository to include the url of the original github repository?

File: ReleaseNotes.md [hunk 2]
```diff
@@ -110,7 +110,8 @@ categorized by license:
 * [jQuery Mockjax v2.2.1](https://github.com/jakerella/jquery-mockjax)
 * [jQuery scrollTo v1.4.2](http://flesler.blogspot.com/2007/10/jqueryscrollto.html)
 * [jQuery Touch Punch v0.2.2](http://touchpunch.furf.com/)
-* [jquery.simulate](https://github.com/eduardolundgren/jquery-simulate)
+* [jquery.simulate v0.5 (forked)](https://github.com/eduardolundgren/jquery-simulate)
```
Comment (cindyli) [inline ReleaseNotes.md]: Did you add ```(forked)``` to the wrong line? Shall it be for ```jquery.selectbox```?

File: tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md (hunk 1 above)
Comment (jobara) [inline tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md]: @cindyli what do you mean by original github repository? If you mean what we forked the code from, that place doesn't seem to exist anymore. We just happened to have the old version of the code in our codebase, so I moved that into a repo under fluid-project.

File: ReleaseNotes.md (hunk 2 above)
Comment (jobara) [inline ReleaseNotes.md]: @cindyli thanks for catching that.. i did. I'll fix that up.

File: tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md (hunk 1 above)
Comment (cindyli) [inline tests/manual-tests/lib/jquery/plugins/selectbox/VERSION.md]: I see.

**Comment by cindyli** [comment]
Merged at a1199e7ff7d856b30d63fd55412534069b3d63d8

**Pull request merged**